src/main/kotlin/gg/skytils/skytilsmod/core/Config.kt

```diff
@@ -3092,6 +3093,9 @@ object Config : Vigilant(
 
         addDependency("assumeWitherImpact", "witherShieldCooldown")
 
+        addDependency("showTrophyFishTotals", "trophyFishTracker")
+        addDependency("showTotalTrophyFish", "trophyFishTracker")
+
```

Comment:
Why is this here?

Reply:
![image](https://github.com/Skytils/SkytilsMod/assets/37018278/a5e223c0-f238-4c1d-858f-1b97d5962455)
??

Reply:
![image](https://github.com/Skytils/SkytilsMod/assets/47618543/f7837457-f7ec-4fb3-8111-813181f59c40)

Reply:
I think it's because you used merge instead of rebase

Reply:
Is everything else fine?

Reply:
Yeah seems fine